dbfit-java/build.gradle

```diff
@@ -387,6 +387,20 @@ project('postgres') {
     }
 }
 
+project('snowflake') {
+    description = 'DBFit snowflake engine'
+
+    dependencies {
+        compile project(':dbfit-java:core')
+        testCompile project(':dbfit-java:core').sourceSets.test.output
+        compile group: 'net.snowflake', name: 'snowflake-jdbc', version: '3.12.15'
```

Comment:
May we move that `compile` 1 line up (next to the other `compile`). And rather change to "group:name:version" syntax as it is a bit more common across the project for now. I can see there is already `3.13.0` released - maybe we can bump up the version?